Parse/src/main/java/com/parse/Parse.java

```diff
@@ -643,37 +629,24 @@ private static ParseEventuallyQueue getEventuallyQueue(Context context) {
     }
   }
 
-  static void checkInit() {
```

Comment:
What's the reason for deleting?  Parse LiveQuery depends on this function to check to see if Parse has been initialized.